### DISCUSSION TITLE
Extend the numbers of items in carousel

### DISCUSSION THREAD

**Discussion by MozeBaltyk** [razonyang/hugo-theme-bootstrap#993] [Ideas]
Hello,
 
Just a feature idea, the possibility to set the numbers of items in the carousel. 
Currently there is 3 articles displayed by the carousel. I did not found a parameter either in the doc either the config files.

That's would be a nice to have. But thanks for your nice work.

**Comment by razonyang** [comment]
LGTM, closed by #994.
